dockerhub/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM idealista/jdk:8u212-stretch-openjdk-headless
+ARG BASE_IMAGE
+FROM idealista/$BASE_IMAGE
```

Comment:
why not set this to `FROM $BASE_IMAGE` and set there the idealista namespace? This way, the Dockerfile isn't coupled to idealista